addons/flannel/0.21.5/install.sh

```diff
@@ -226,7 +226,28 @@ function weave_to_flannel() {
 
     logStep "Restarting kubelet"
     systemctl stop kubelet
-    iptables -t nat -F && iptables -t mangle -F && iptables -F && iptables -X
+
+    logSubstep "Flushing and deleting Weave entries from IP tables"
+    # Array of iptables tables. Add more if required.
+    tables=("filter" "nat" "mangle")
+
+    for table in "${tables[@]}"; do
+        # Get all chains for the current table
+        chains=$(iptables -t $table -n -L | grep 'Chain WEAVE-' | awk '{print $2}')
+
+        for chain in $chains; do
+            # Flush and delete each chain
+            echo "Flushing and deleting $chain from $table table"
+            iptables -t $table -F $chain
+            iptables -t $table -X $chain
+        done
+    done
+    # Display the current iptables rules
+    echo "Current iptables rules:"
+    echo "$(iptables -L)"
+
+    logSuccess "Weave successfully flushed from iptables"
```

Comment:
If you're going to log this, make sure that you spell iptables the same way each place
see
```
    logSubstep "Flushing and deleting Weave entries from IP tables"
```
above